docs/sdks/server-sdks/node.md

```diff
@@ -31,40 +31,11 @@ npm install @eppo/node-server-sdk
 </TabItem>
 </Tabs>
 
-## 2. Initialize the SDK
```

Comment:
What do you think of wrapping sections 1-5 in a Getting Started section? 

So the right sidebar would look like this:

```
Getting Started
  Install the SDK
  Create logger
  Initialize the SDK
  Getting variants
  Example
Handling NULL
Typed assignments
Initialization options
```

Reply:
I agree -- updated this